src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/vendored_sdks/azure_mgmt_cosmosdb/models/restorable_mongodb_collection_get_result.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from .arm_resource_properties import ARMResourceProperties
+
+
+class RestorableMongodbCollectionGetResult(ARMResourceProperties):
+    """An Azure Cosmos DB restorable MongoDB collection.
+
+    Variables are only populated by the server, and will be ignored when
+    sending a request.
+
+    :ivar id: The unique resource identifier of the ARM resource.
+    :vartype id: str
+    :ivar name: The name of the ARM resource.
+    :vartype name: str
+    :ivar type: The type of Azure resource.
+    :vartype type: str
+    :param location: The location of the resource group to which the resource
+     belongs.
+    :type location: str
+    :param tags:
+    :type tags: dict[str, str]
+    :param identity:
+    :type identity: ~azure.mgmt.cosmosdb.models.ManagedServiceIdentity
+    :param resource:
```

Comment:
I have commented the same on Vinh's sdk-for-net-pr. I believe we have added "allof: ARMProperties" wrongly in swagger. I dont think restorable resources have these properties.

Reply:
As discussed offline, I will update this in the next iteration of swagger changes